ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -247,30 +278,38 @@ def verify_dut_lag_member_state(self, lag_memb_output, pre_check=True):
         return success
 
     def verify_dut_lag_state(self, pre_check=True):
-        pat = re.compile(".*%s\s+LACP\(A\)\(Dw\)\s+(.*)" % self.vm_dut_map[self.neigh_name]['dut_portchannel'])
+        pat = re.compile("\s+\d+\s+(\w+\d+)\s+LACP\(A\)\(Dw\)\s+(.*)")
```

Comment:
>"\s+\d+\s+(\w+\d+)\s+LACP\(A\)\(Dw\)\s+(.*)" [](start = 25, length = 44)

Could you give matched line sample, and what do you want to extract?

Reply:
Matched line example: 0001  PortChannel0001  LACP(A)(Dw)  Ethernet0(D) Ethernet4(S)
I will extract the portchannel name and members and will do a comparison for members in a separate func based on what state I expect for each member

Reply:
Better to comment code.

Reply:
added a comment